app/views/curation_concerns/base/_member.html.erb

```diff
@@ -0,0 +1,16 @@
+<tr class="<%= dom_class(member) %> attributes">
+  <td class="thumbnail">
+    <% if member.reader? %>
```

Comment:
Ah, that's a good call. @jcoyne @tpendragon any chance y'all can find time for this today? Can this solution work for 7.3.0? (We can ticket further optimization in Hyrax if needed.)